templates/repo/issue/labels/labels_selector_field.tmpl

```diff
@@ -37,7 +36,7 @@
 				{{if .Description}}<br><small class="desc">{{.Description | RenderEmoji $.Context}}</small>{{end}}</a>
 			{{end}}
 		{{else}}
-			<div class="header" style="text-transform: none;font-size:14px;">{{.locale.Tr "repo.issues.new.no_items"}}</div>
+			<div class="header header-no-uppercase">{{.locale.Tr "repo.issues.new.no_items"}}</div>
```

Comment:
According to official document, it should be a "disabled item":

![image](https://github.com/go-gitea/gitea/assets/2114189/4b69c50f-f7f2-4a3a-8e28-d0a01f58b92c)

Reply:
Ah yes, much better than abusinga header.